CodeGeneration/Sources/SyntaxSupport/StmtNodes.swift

```diff
@@ -440,7 +440,10 @@ public let STMT_NODES: [Node] = [
       Child(
         name: "BindingSpecifier",
         deprecatedName: "BindingKeyword",
-        kind: .token(choices: [.keyword(text: "let"), .keyword(text: "var"), .keyword(text: "inout")])
+        kind: .token(choices: [
+          .keyword(text: "let"), .keyword(text: "var"), .keyword(text: "inout"),
+          .keyword(text: "mutating"), .keyword(text: "borrowing"), .keyword(text: "consuming"),
+        ])
```

Comment:
AFAICT we don’t have tests for these. Could you add some? Should be `if mutating x = x {}`